app/Helpers/database/achievement.php

```diff
@@ -65,9 +65,9 @@ function getAchievementsList(
         $query .= "AND pa.unlocked_at IS NULL ";
     }
 
-    if (isValidUsername($developer)) {
-        $bindings['author'] = $developer;
-        $query .= "AND ach.Author = :author ";
+    if (isValidUsername($developer?->username)) {
```

Comment:
This can probably just be `if ($developer)`. I wouldn't expect an invalid username to resolve into a model.

Reply:
Yep, good point. Fixed in latest.